File: java/Vehicle.java
public class Vehicle {
    private String brand;
    private int year;

    public Vehicle() {
        this("Unknown", 0);
    }

    public Vehicle(String brand, int year) {
        this.brand = brand;
        this.year = year;
    }

    public String getBrand() {
        return brand;
    }

    public void setBrand(String brand) {
        this.brand = brand;
    }

    public int getYear() {
        return year;
    }

    public void setYear(int year) {
        this.year = year;
    }

    public void displayInfo() {
        System.out.println("Brand " + brand);
        System.out.println("Year " + year);
    }

}

File: python/26/2/Vehicle.py
class Vehicle:
    def __init__(self,brand="Unknown",year=0) :
        self.__brand=brand
        self.__year=year
    def getBrand(self):
        return self.__brand
    def getYear(self):
        return self.__year
    def setBrand(self,brand):
        self.__brand=brand
    def setYear(self,year):
        self.__year=year
    def displayInfo(self):
        print("Brand: ",self.__brand)
        print("Year: ",self.__year)
